File: bilgi-yarismasi-spa/src/app/models/question.ts
export class Question {
    id:number;
    questionText:string;
    choice1:string;
    choice2:string;
    choice3:string;
    choice4:string;
    answer:string;

    constructor(Id,QuestionText,Choice1,Choice2,Choice3,Choice4,Answer){
        this.id=Id;
        this.questionText=QuestionText;
        this.choice1=Choice1;
        this.choice2=Choice2;
        this.choice3=Choice3;
        this.choice4=Choice4;
        this.answer=Answer;
    }
}

File: BilgiYarismasi.API/BilgiYarismasi.API/Models/Question.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BilgiYarismasi.API.Models
{
    public class Question
    {
        public int Id { get; set; }

        public string QuestionText { get; set; }

        public string Choice1 { get; set; }

        public string Choice2 { get; set; }

        public string Choice3 { get; set; }

        public string Choice4 { get; set; }

        public string Answer { get; set; }


    }
}
